src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -1,7 +1,7 @@
 use vs
 
 package name=Microsoft.VisualStudio.Editors
-        version=15.0.0.0
+        version=$(Version)
```

Comment:
@basoundr is version here 15.0.0.x? I think one of the issues is that some components are versioned 2.0.x while this component was versioned 15.0.x